packages/cli/src/utils.ts

```diff
@@ -314,6 +357,11 @@ export function getLocalTeamsAppId(rootfolder: string | undefined): any {
   }
 
   if (isWorkspaceSupported(rootfolder)) {
+    // TODO: read local teams app ID from localSettings.json instead of env file
+    if (isMultiEnvEnabled()) {
+      return undefined;
+    }
```

Comment:
Will you handle reading local Teams app ID from `localSettings.json`? otherwise it will break local debug if multi-env is enabled.

Reply:
It won't break local debug in CLI because this function is only used for telemetry. The sideloading teams app id is retrieve by `executeUserTask`

Reply:
I see. There's another `getLocalTeamsAppId` in `preview.ts`, which is used to preview a local Teams app. Can we consolidate this two APIs into one, since this is only use for local preview?

Reply:
In my understanding,
- `getLocalTeamsAppId()` is only for local debug. And this is the localdebug teams app id.
- `getTeamsAppId()` is for all telemetry and this is the remote appId. Other operations like provision, deploy, etc. can be linked together with this appid in telemetry. In multi-env scenarios, we need to use the current active env teamsapp id.

They are not the same thing. So I don't think we should merge them into one function, because I think one function should do one thing.

Reply:
Both these two APIs are `getLocalTeamsAppId()`, that's why I'm considering merge these tow APIs into one. 

![image](https://user-images.githubusercontent.com/10163840/132790265-1962d665-9a16-476f-990b-4e7392c01726.png)

![image](https://user-images.githubusercontent.com/10163840/132790415-774cc846-fe2e-4a4e-8273-977a601a0a64.png)

`

Reply:
Sorry, I forgot to expand the code on github. You are correct. I will refactor later.

Reply:
We can refine this in a another PR~BTW, you can use the Microsoft CodeFlow tool to do code review, which will be more clearer than GitHub :).

Reply:
Thanks for the advice~